modules/nf-core/bbmap/bbnorm/meta.yml

```
@@ -0,0 +1,43 @@
+name: "bbmap_bbnorm"
+description: BBNorm is designed to normalize coverage by down-sampling reads over high-depth areas of a genome, to result in a flat coverage distribution. 
+keywords:
+  - normalization
+  - assembly
+
+tools:
+  - "bbmap":
+      description: "BBMap is a short read aligner, as well as various other bioinformatic tools."
+      homepage: "https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/"
+      documentation: "https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/"
+      tool_dev_url: "https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbnorm-guide/"
+      doi: ""
+      licence: "BBMap - Bushnell B. - sourceforge.net/projects/bbmap/"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - fastq:
+      type: file
+      description: fastq file
+      pattern: "*.{fastq,fq}"
```

```suggestion
      pattern: "*.{fastq,fq}(.gz)?"
```
I'm unsure of the `(.gz)?` since I don't know if this is a regexp or a glob. Maybe you can find examples in other modules. The idea is that gzipped files should be accepted but not required.